sherlock/document.py

```diff
@@ -152,6 +191,26 @@ def from_dict(cls, doc: "Document", dct: Dict[str, Any]) -> "Mention":
 
 @dataclass(frozen=True)
 class Entity:
+    """
+    Representation for an Entity class within a Document.
+
+    An Entity is an abstract Category for (groups of) Tokens,
```

Comment:
better: "An entity describes a unique, real-world concept, such as a Person, that is referred to by a group of Mentions within this document as well as a list of reference ids in external knowledge bases, such as Wikidata or Freebase. "